cmd/root.go

```diff
@@ -41,8 +41,12 @@ It can be used to export, import or sync entities to Kong.`,
 // Execute adds all child commands to the root command and sets
 // sflags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
-func Execute() error {
-	return rootCmd.Execute()
+func Execute() {
```

Comment:
Did you consider setting `cobra.Command`'s `SilenceErrors` to `true` instead?

Reply:
I did. I reverted to match the past to avoid further new surprises.